connect-widget/app/src/shared/components/WidgetContainer.tsx

```diff
@@ -56,6 +57,33 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
     },
   });
 
+  const [displayIcon, setDisplayIcon] = useState(true);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerHeight < 1050) {
```

Comment:
is this the best way to do this?

whenever pixel are hardcoded it usually means that it won't work for all screen sizes

Reply:
Fixed